docs/CHANGELOG.asciidoc

```diff
@@ -89,6 +90,17 @@ This section summarizes the changes in each release.
 //=== Known Issues
 ////
 
+[[release-notes-6.8.3]]
+== {kib} 6.8.3
+
+[float]
+[[bug-6.8.3]]
+=== Bug fixes
+Monitoring::
+* Fixes the inability to complete a single `date_histogram` aggregation for `get_nodes` calls {pull}43481[#43481]
+Operations::
+* Adds the `"org.label-schema.license"` Docker image label to override the license field from the baseimage {pull}42836[#42836]
```

Comment:
I am not sure what the user impact of this change is - we may want to have a more user-facing description....

Reply:
No need to mention this PR in the RN.